apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -6,12 +6,16 @@ import Button from '../Button';
 import i18n from '@cdo/locale';
 import firehoseClient from '@cdo/apps/lib/util/firehose';
 
+const PASSWORD_TOO_SHORT_ERROR_MESSAGE =
```

Comment:
Maybe add a comment indicating that this is the message sent on validation error by the back-end?